tools/issue-labeler/src/IssueLabelerService/Configuration.cs

```diff
@@ -1,6 +1,6 @@
 using Microsoft.Extensions.Configuration;
 
-namespace ConfigurationService
+namespace IssueLabelerService
```

Comment:
Can we add folders for organization.   It would be nice to have:
   - Configuration
   - Labels
   - Answers

Feel free to tune the text, but I'm thinking the discrete areas that we'd want to modularize should each have a folder.